molecular/src/main/kotlin/com/hartwig/actin/molecular/panel/PanelEvidenceAnnotator.kt

```diff
@@ -0,0 +1,126 @@
+package com.hartwig.actin.molecular.panel
+
+import com.hartwig.actin.datamodel.molecular.PanelRecord
+import com.hartwig.actin.datamodel.molecular.characteristics.MolecularCharacteristics
+import com.hartwig.actin.datamodel.molecular.driver.CopyNumber
+import com.hartwig.actin.datamodel.molecular.driver.DriverLikelihood
+import com.hartwig.actin.datamodel.molecular.driver.Fusion
+import com.hartwig.actin.datamodel.molecular.driver.Variant
+import com.hartwig.actin.molecular.MolecularAnnotator
+import com.hartwig.actin.molecular.driverlikelihood.GeneDriverLikelihoodModel
+import com.hartwig.actin.molecular.evidence.EvidenceDatabase
+import com.hartwig.actin.molecular.interpretation.GeneAlterationFactory
+
+class PanelEvidenceAnnotator(
+    private val evidenceDatabase: EvidenceDatabase,
+    private val geneDriverLikelihoodModel: GeneDriverLikelihoodModel,
+) : MolecularAnnotator<PanelRecord, PanelRecord> {
+
+    override fun annotate(input: PanelRecord): PanelRecord {
+        return input
+            .let(::annotateWithDriverAttributes)
```

Comment:
I like how you've centralized all annotations requiring the evidence database, but this code makes me think we could do one more split, as conceptually I guess driver attributes are not evidence, and as such it might lead to some confusion trying to find this code, or worse getting the order mixed up.

I guess this is causes by evidence database being more than just an evidence database, but I think refactoring that is not necessary at this point. 

What do you think about having a PanelDriverAttributeAnnotator and a PanelEvidenceAnnotator?